packages/renderer/src/lib/troubleshooting/TroubleshootingPageStoreDetails.svelte

```diff
@@ -39,33 +41,24 @@ async function fetch(): Promise<void> {
         <div class="bg-charcoal-800 w-full p-4 mb-4">
           <div class="pb-2 text-lg">Info:</div>
 
-          <div class="mx-2 flex flex-row items-center">
-            Size: <div role="status" aria-label="size">{eventStoreInfo.size}</div>
-            <div class="mx-2">
-              <button
-                disabled="{fetchInProgress}"
-                class="px-3 my-1 text-sm font-medium text-center text-white bg-violet-600 rounded-sm hover:bg-dustypurple-800 focus:ring-2 focus:outline-none focus:ring-dustypurple-700"
-                title="Refresh"
-                on:click="{() => fetch()}">
-                Refresh
-              </button>
-            </div>
+          <div class="mx-2 flex flex-row items-center space-x-2">
+            Size:&nbsp;
```

Comment:
why &nbsp; instead of using the mx-2 ?
also it should be part of another PR as it's not related to buttons

Reply:
I assume the character I can't see is a `nbsp;`? ;-)
It was showing up as`Size:1` without a space so I was just forcing one in there. Since it is not related to buttons I removed in new commit though.